docs/getting-started.md

```diff
@@ -74,11 +74,61 @@ instructions for downloading static binaries that are published with every relea
 
 We provide [Tanka](https://tanka.dev) configurations in our [`production/`](/production/tanka/grafana-agent) directory.
 
-## Migrating from Prometheus
+## Creating a Config File
 
-Migrating from Prometheus is relatively painless, requiring just copying and
-pasting sections from the Prometheus configuration file into the Agent
-configuration file. Use this Agent config as a template:
+The Grafana Cloud Agent can be configured with **integrations** or a
```

Comment:
integrations _or_ prometheus config reads as an exclusive-or to me. How about "and"?